From 76e28fc006ee42f88f608d44866e683cf2244ff8 Mon Sep 17 00:00:00 2001
From: dann frazier <dann.frazier@chainguard.dev>
Date: Tue, 15 Sep 2026 11:33:31 -0600
Subject: [PATCH 02/18] Revert "tests: env: fix a false failure on GNU/Hurd"

This reverts commit 1859316a9fbae386ae438eb819be4ddfa431c465.
NEWS left untouched.
---
 tests/env/env.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/env/env.sh b/tests/env/env.sh
index be822e346..b00402deb 100755
--- a/tests/env/env.sh
+++ b/tests/env/env.sh
@@ -204,8 +204,7 @@ tr "'" '"' <exp-noargs-shell >exp-noargs-c || framework_failure_
 for qs in literal shell c; do
   env -i PATH="$PATH" QUOTING_STYLE=$qs 'a b'='c d' \
     env >out-t 2>err || fail=1
-  grep -vE '^["'"'"']?'\
-'(__CF_USER_TEXT_ENCODING|QUOTING_STYLE|(LD_ORIGIN_)?PATH)["'"'"']?=' \
+  grep -vE '^["'"'"']?(__CF_USER_TEXT_ENCODING|QUOTING_STYLE|PATH)["'"'"']?=' \
     out-t >out || framework_failure_
   compare exp-noargs-$qs out || fail=1
   compare /dev/null err || fail=1
-- 
2.53.0

